packages/modular-scripts/src/esbuild-scripts/config/createEsbuildConfig.ts

```diff
@@ -65,7 +67,7 @@ export default function createEsbuildConfig(
     color: !isCi,
     define,
     metafile: true,
-    tsconfig: paths.appTsConfig,
+    tsconfig: isApp ? paths.appTsConfig : undefined,
```

Comment:
I don't think we actually need tsconfig files for any app / view / etc and this is just a relic of CRA. We could probably remove this now?

Reply:
Done.